src/main/java/com/google/alpollo/Song.java

```diff
@@ -0,0 +1,25 @@
+package com.google.alpollo;
+
+public class Song {
```

Comment:
Can you please add documentation to this and the other classes added in this PR?